frontend/public/components/factory/Table/VirtualizedTable.tsx

```diff
@@ -181,6 +183,29 @@ const VirtualizedTable: VirtualizedTableFC = ({
     </WindowScroller>
   );
 
+  const formatToCsv = (array, delimiter = ',') =>
+    array
+      .map((row) =>
+        row.map((rowItem) => (isNaN(rowItem) ? `"${rowItem}"` : rowItem)).join(delimiter),
```

Comment:
Do we need to do this validation here, or should be better to leave this responsibility to the consumers. It seems they should pass the data ready in CSV including numeric formatting or transformation, WDYT?

Reply:
That makes sense. I'll remove the formatToCsv() function and so the consumer handles this.

Reply:
TODO: Double check that there is data in the csvData object

Reply:
`csvData` is a string, so if it is an empty string, null, or undefined the condition is false and the <Button> will not render. 

```
{csvData && (
              <Button className="co-virtualized-table--export-csv-button" onClick={downloadCsv}>
                Export as CSV
              </Button>
            )}
```